ITensorGaussianMPS/examples/spinless_fermion_pairing.jl

```diff
@@ -49,16 +49,22 @@ let
   H = ITensors.MPO(os_h + os_p, sites)
 
   #Get Ground state 
-  E, V = eigen(Hermitian(h))
-  @show E
+  @assert ishermitian(h)
+  e = eigvals(Hermitian(h))
+  @show e
+  E, V = gaussian_eigen(h)
+  @show sum(E[1:N])
   Φ = V[:, 1:N]
   c = conj(Φ) * transpose(Φ)
```

Comment:
I would put the call to `real` here, i.e. `c = real(conj(Φ) * transpose(Φ))` to emphasize that step more.